guides/common/modules/con_api-request-composition.adoc

```diff
@@ -11,15 +12,16 @@ To work with the API, construct a command by using the `curl` command syntax and
 
 [options="nowrap", subs="+quotes,attributes"]
 ----
-$ curl --request _HTTP_METHOD_ \         #<1>
---insecure \                           #<2>
---user _My_User_Name_:__My_Password__ \      #<3>
---data @_input_file_.json \              #<4>
+$ curl \
+--request _HTTP_METHOD_ \                  #<1>
+--insecure \                               #<2>
+--user _My_User_Name_:__My_Password__ \    #<3>
+--data @_My_Input_File_.json \             #<4>
 --header "Accept:application/json" \       #<5>
 --header "Content-Type:application/json" \ #<5>
---output _output_file_                    #<6>
-_API_ROUTE_ \                             #<7>
-| python3 -m json.tool                  #<8>
+--output _My_Output_File_                  #<6>
+_API_ROUTE_ \                              #<7>
+| python3 -m json.tool                     #<8>
```

Comment:
I'm not sure about changing the spacing before the numbers. Do you want it to look good in the build or in the source? You can't have both :))

Reply:
Fixed and added a comment.